### ISSUE TITLE
Could you release the training code?

### ISSUE THREAD

**Issue by TPZZZ** [lingyanruan/DRBNet#5]
Dear author,

Congratulations on your article being accepted as the oral paper of cvpr.

In your paper. the most impressive result is in Table 5, when you replace the L1 loss with Lvgg loss in stage 2 to train your model, the performance on Realdof dataset is significantly increased and the performance of DPDD is decreased subtlely.

I follow your scheme that firstly trains our model on LF dataset  and then fine-tune our model on DPDD dataset with Lvgg loss.

But I can not such high performance improvement on RealDof dataset when using Lvgg loss in the stage 2 and even get worse performance on both datasets when compare to using L1 loss in the stage 2. So could release the training code ? In this case, we can see if it is the error caused by our Lvgg loss or your scheme is only useful to your model . 

Because you tried to use the scheme to train the DPDNet and KPAC, but only showed the results on DPDD dataset and did not show the results on RealDof dataset.
Furthermore , I would like to ask that did you test to use L1 loss to fine-tune DPDNet and KPAC in the stage 2 to see if their results are better or worse than that they use L_vgg loss for fine-tuning.

**Comment by lingyanruan** [comment]
Hi,

 I am not available to release the training code for now due to my tight schedule. I will try to release it later when I grab some free time. Here I could give you some suggestions to reproduce our training.

'But I can not such high performance improvement on RealDof dataset when using Lvgg loss in the stage 2 and even get worse performance on both datasets when compare to using L1 loss in the stage 2.''

Please be aware of the learning rate in second stage, in our paper we set it to 1e-5 and then linearly decay to the end. The second stage may need different amount of iterations depending on the network structure. Usually the best performance doesn't happen in the end. You may refer to the early stage like 10~50 epoch for example.  
Please note that use the 350 images for one epoch training rather than cropped 7k images like DPDNet did.

I believe this training scheme could be beneficial to other cnn networks (KPAC, DPDNet) as reported in our paper.

**Comment by TPZZZ** [comment]
Thank you for your so quick reply and I will try.

Best.
